postgres_replicator/transfer.go

```diff
@@ -0,0 +1,228 @@
+package transfer
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/leporo/sqlf"
```

Comment:
why was this library chosen? github.com/leporo/sqlf
Without more context it seems arbitrary.

Reply:
I need to build SQL queries programatically based on arbitrary table schema. More popular SQL builder libraries seemed to lack this feature. I've also added this information to the design document.